readme.md

```diff
@@ -129,6 +129,7 @@ Here you find projects that were made especially for hacktoberfest.
 - [Gatsby](https://github.com/gatsbyjs/store.gatsbyjs.org)
 - [Grafana Labs](https://github.com/grafana/grafana)
 - [Hasura](https://blog.hasura.io/hasura-joins-hacktoberfest-2019/)
+- [JetBrains](https://www.jetbrains.com/lp/hacktoberfest-2020/)-JetBrains offers a wide range of developer IDEs. For Hacktoberfest, it makes them available for free for open source contributors independently of the contributor being a permanent contributor to an open source project (which is covered by their Free Licenses Program)
```

Comment:
I checked OUR contribution guide. It states:

> Add a description to your proposal. See sindresorhus/awesome@master/pull_request_template.md for details.

This file contains examples. I think, we are following more or less these examples.

However, did you see that there is a space before and after the dash? Can you please add the two spaces?

I also forgot the final dot at the sentence.

```suggestion
- [JetBrains](https://www.jetbrains.com/lp/hacktoberfest-2020/) - JetBrains offers a wide range of developer IDEs. For Hacktoberfest, it makes them available for free for open source contributors independently of the contributor being a permanent contributor to an open source project (which is covered by their Free Licenses Program).
```